src/governance.cpp

```diff
@@ -744,8 +738,7 @@ void CGovernanceManager::SyncAll(CNode* pnode, CConnman& connman) const
 
     CNetMsgMaker msgMaker(pnode->GetSendVersion());
     connman.PushMessage(pnode, msgMaker.Make(NetMsgType::SYNCSTATUSCOUNT, MASTERNODE_SYNC_GOVOBJ, nObjCount));
-    connman.PushMessage(pnode, msgMaker.Make(NetMsgType::SYNCSTATUSCOUNT, MASTERNODE_SYNC_GOVOBJ_VOTE, nVoteCount));
-    LogPrintf("CGovernanceManager::%s -- sent %d objects and %d votes to peer=%d\n", __func__, nObjCount, nVoteCount, pnode->id);
+    LogPrintf("CGovernanceManager::%s -- sent %d objects and %d votes to peer=%d\n", __func__, nObjCount, pnode->id);
```

Comment:
Looks like this log string has too many conversion specifiers now -- should remove ` and %d votes` maybe?

Reply:
FYI, as this struck me maybe a dozen times and the logging functions tend to crash the whole app in such cases (without meaningful error messages), I've got a few changes in one of my local branches which avoid a crash and print which format string cased the problems. Gonna include this in one of my future PRs or a separate PR.